gammapy/modeling/fit.py

```diff
@@ -895,3 +897,69 @@ def _repr_html_(self):
             return self.to_html()
         except AttributeError:
             return f"<pre>{html.escape(str(self))}</pre>"
+
+
+class FitResults(collections.abc.Sequence):
```

Comment:
if you use `collections.abc.MutableSequence` instead, then `append` etc should work automatically, no